### DISCUSSION TITLE
Exit Transitions with v. 13

### DISCUSSION THREAD

**Discussion by kerns** [vercel/next.js#44136] [Help]
I am new to next.js and would like to learn using the new conventions in 13 as I work on projects that are not mission critical. Please note I realize 100% that the app folder approach is experimental so I'm _not_ complaining, just trying to understand where the issue is. :)

I have noticed consistently across the 4 or 5 tutorials I've followed is that exit animations don't seem achievable, at least with Framer Motion – but if this is a shortcoming of Framer Motion and `AnimatePresence`, v. 13 when using the app folder approach, or v. 13 broadly I don't actually know.

Just wondering if anyone else was tracking this functionality and where the issue was? I have considered trying GSAP and react router instead... 

Anyone using the app folder approach AND have exit transitions operational? Input welcome.

**Comment by khuezy** [comment]
I got exit animation using appDir and latest 13.2.2-canary.1 here: https://github.com/framer/motion/issues/1850#issuecomment-1445239322
